language_names/manual.py

```diff
@@ -0,0 +1,32 @@
+import sqlite3
+import textwrap
+import csv
+
+
+def populate_database():
+    conn = sqlite3.connect('test.db')
+    c = conn.cursor()
+    c.execute(textwrap.dedent("""
```

Comment:
This SQL statement and the one below doesn't really need dedent.

Reply:
okay I will verify